packages/devtools-source-map/src/source-map.js

```diff
@@ -5,7 +5,7 @@
  * @module utils/source-map-worker
  */
 
-const networkRequest = require("devtools-network-request");
+const { networkRequest } = require("devtools-modules");
```

Comment:
this replaces the `devtools-network-request` with `devtools-modules`. I'll update the webpack.config.devtools in a minute to reflect this